### ISSUE TITLE
@LogClass does not work with `type-graphql`

### ISSUE THREAD

**Issue by vickylance** [aigoncharov/class-logger#7]
I am getting this error when using with `type-graphql`

```cmd
(node:66568) UnhandledPromiseRejectionWarning: Error: Generating schema error
    at Function.generateFromMetadata (/Volumes/Projects/flutter/tw_foodies/server/node_modules/type-graphql/dist/schema/schema-generator.js:19:23)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at Object.buildSchema (/Volumes/Projects/flutter/tw_foodies/server/node_modules/type-graphql/dist/utils/buildSchema.js:9:20)
    at /Volumes/Projects/flutter/tw_foodies/server/src/index.ts:162:15
(node:66568) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:66568) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

This is how I am using.

```ts
@LogClass()
@Resolver(() => Rating)
export class RatingResolver {
  @Log()
  @Mutation(() => Boolean)
  public async addRating(
    @Arg("rating", () => Float) rating: number,
    @Arg("review", () => String, { nullable: true }) review: string | null
  ): Promise<Boolean> {
    try {
      await Rating.insert({ rating, review });
      return true;
    } catch (err) {
      throw err;
    }
  }
  @Log()
  @Mutation(() => Boolean)
  public async deleteRating(@Arg("id", () => ID) id: number): Promise<Boolean> {
    try {
      await Rating.delete({ id });
      return true;
    } catch (err) {
      throw err;
    }
  }
}
```

**Comment by vickylance** [comment]
I think this package does not work when there are other decorators

**Comment by aigoncharov** [comment]
Should be fixed in 1.3.0. Could you try it and reopen the issue if necessary?
